styles.css

```diff
@@ -103,6 +103,243 @@ header {
   list-style-type: none;
 }
 
+/* Work Section */
+.work {
+  margin-top: 5rem;
+  padding-bottom: 5rem;
+}
+
+.work .heading {
+  text-align: center;
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 400;
+  font-style: normal;
+  font-family: 'Crete Round', serif;
+  margin: 0 auto 2rem auto;
+  color: #1c1a19;
+}
+
+.line {
+  margin: 1rem 1rem 4rem;
```

Comment:
To match your template design, kindly increase the margin right/left to be at least `2rem` :+1:

Reply:
Done